usr/share/rear/layout/save/GNU/Linux/220_lvm_layout.sh

```diff
@@ -84,25 +85,38 @@ Log "Saving LVM layout."
             chunksize="$(echo "$line" | awk -F ':' '{ print $7 }')"
             stripes="$(echo "$line" | awk -F ':' '{ print $8 }')"
             stripesize="$(echo "$line" | awk -F ':' '{ print $9 }')"
+            segmentsize="$(echo "$line" | awk -F ':' '{ print $10 }')"
 
             kval=""
             [ -z "$thinpool" ] || kval="${kval:+$kval }thinpool:$thinpool"
             [ $chunksize -eq 0 ] || kval="${kval:+$kval }chunksize:${chunksize}b"
             [ $stripesize -eq 0 ] || kval="${kval:+$kval }stripesize:${stripesize}b"
+            [ $segmentsize -eq $size ] || kval="${kval:+$kval }segmentsize:${segmentsize}b"
```

Comment:
@rmetrich to me it seems like an abuse of the `kval` variable to put `segmentsize` into it, since unlike the others it is not a valid option for lvcreate. I thus submitted #2196 to change this, if you agree.

Reply:
Huh, what? Abuse?
OK, I recognize I was lazy here :-) Thanks for the PR.